scripts/commands/updateApiDocs.ts

```diff
@@ -50,6 +50,11 @@ interface Arguments {
   packages: string[];
 }
 
+interface Link {
+  url: string;
+  text?: string;
```

Comment:
Maybe `description`? `text` is a little vague.

Reply:
I think `text` is the correct term, but I've added a comment in [3d0e8b2](https://github.com/Qiskit/documentation/pull/312/commits/3d0e8b24e5dd65ce0e921f347afeb485f43f2307) to clarify what it means.